### PR TITLE
Feature/df 1568 delete competence set

### PR DESCRIPTION
**Pull request by emina-str**
translations will come in couple of days ...

### CODE REVIEW

File: alv-portal-ui/translations.csv [hunk 1]
```diff
@@ -1188,6 +1189,7 @@ portal.competence-catalog.competence-elements.add-modal.added-success-notificati
 portal.competence-catalog.competence-elements.filter-modal.title,Kompetenz-Bestandteile filtern,Filter skill components,Filtrer les composants de compétence,Filtrare elementi della competenza,
 portal.competence-catalog.competence-sets.actions.search-and-add,Suchen & Hinzufügen,Search and add,Rechercher et ajouter,Cercare & aggiungere,
 portal.competence-catalog.competence-sets.actions.unlink,Verknüpfung aufheben,Remove link,Annuler la connexion,Sopprimere collegamento,
+portal.competence-catalog.competence-sets.overview.delete.confirm,"Ja, löschen","Yes, delete","Oui, supprimer","Sì, cancellare",
```
Comment (cjpsycos) [inline alv-portal-ui/translations.csv]: would name it portal.global.delete.confirm

File: alv-portal-ui/translations.csv [hunk 2]
```diff
@@ -1200,6 +1202,11 @@ portal.competence-catalog.competence-sets.edit-button.tooltip,Bearbeiten,Edit,É
...
 portal.competence-catalog.competence-sets.added-competence-element-success-notification,Bestandteil hinzugefügt,Component added,Composant ajouté,Componente aggiunto,
 portal.competence-catalog.competence-sets.removed-competence-element-success-notification,Bestandteil entfernt,Component removed,Composant annulé,Componente rimosso,
+portal.competence-catalog.competence-sets.deleted-success-notification,Kompetenz-Set wurde erfolgreich gelöscht.,,,,
+portal.competence-catalog.competence-set.delete-modal.title,Kompetenz-Set löschen?,,,,
+portal.competence-catalog.competence-set.delete-modal.reference-found-alert,Um dieses Kompetenz-Set zu löschen heben Sie zuerst alle Verknüpfungen von diesem Kompetenz-Set auf.,,,,
+portal.competence-catalog.competence-set.delete-modal.reference-found,Kompetenz-Set wird in folgenden CH Fiches verwendet:,,,,
+portal.competence-catalog.competence-set.delete-modal.confirmation,"Möchten Sie dieses Kompetenz-Set ""{{ competenceSetId }}"" wirklich löschen? Wenn Sie auf ""Ja, löschen"" klicken, wird diese Kompetenz-Set unwiderruflich gelöscht.",,,,
```
Comment (cjpsycos) [inline alv-portal-ui/translations.csv]: missing <br>

File: alv-portal-ui/translations.csv (hunk 1 above)
Comment (emina-str) [reply alv-portal-ui/translations.csv]: done

File: alv-portal-ui/translations.csv (hunk 2 above)
Comment (emina-str) [reply alv-portal-ui/translations.csv]: fixed

File: alv-portal-ui/src/app/competence-catalog/competence-sets/competence-set-delete-modal/competence-set-delete-modal.component.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<ng-container *ngIf="{chFiches: chFiches$ | async} as observables">
```
Comment (birom) [inline alv-portal-ui/src/app/competence-catalog/competence-sets/competence-set-delete-modal/competence-set-delete-modal.component.html]: I reopened the https://alv-ch.atlassian.net/browse/DF-927 technical debt issue.

IMHO the following would be a cleaner solution:
```html
<ng-container *ngIf="(chFiches$ | async) as chFiches">
    <alv-modal (cancelAction)="cancel()"
             (primaryAction)="submit()"
             [showPrimaryButton]="!chFiches.length"
             cancelLabel="entity.action.cancel"
             modalTitle="portal.competence-catalog.competence-set.delete-modal.title"
             primaryLabel="portal.global.delete-confirm">

     <ng-container *ngIf="!!chFiches.length; else noReferences">
      <alv-alert [hideDismiss]="true"
                 [notification]=warning>
      </alv-alert>
```

File: alv-portal-ui/src/app/competence-catalog/competence-sets/competence-set-delete-modal/competence-set-delete-modal.component.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<ng-container *ngIf="(chFiches$ | async) as chFiches">
+  <alv-modal (cancelAction)="cancel()"
+             (primaryAction)="submit()"
+             [showPrimaryButton]="!chFiches?.length"
```
Comment (birom) [inline alv-portal-ui/src/app/competence-catalog/competence-sets/competence-set-delete-modal/competence-set-delete-modal.component.html]: ``?.`` can be replaced with ``.``.

**Pull request merged**